Dockerfile.base-browser

```
@@ -0,0 +1,33 @@
+
+LABEL maintainer="Natural-Language-to-Robot-Framework"
+LABEL description="Optimized base image with Chrome, Playwright, UV package manager"
+
+WORKDIR /app
+
+ENV PYTHONUNBUFFERED=1 \
+    PYTHONDONTWRITEBYTECODE=1 \
+    PYTHONIOENCODING=utf-8 \
+    PLAYWRIGHT_BROWSERS_PATH=/root/.cache/ms-playwright
+
+RUN pip install --no-cache-dir uv
```

Hadolint (2.14.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)